README.md

```diff
@@ -190,6 +190,14 @@ object as a parameter to the `copy()` method:
 s4 = s3.copy({store_data: false, bucket_precision: 10 });
 ```
 
+### Getting the raw data
+
+The `data` instance member returns the raw numbers stored in the current Stats object.
```

Comment:
You'll want to mention that this is only true if faststats is configured to store data.

Reply:
done! 👍